homeassistant/components/synology_dsm/camera.py

```diff
@@ -89,6 +92,7 @@ def __init__(
         self.snapshot_quality = api._entry.options.get(
             CONF_SNAPSHOT_QUALITY, DEFAULT_SNAPSHOT_QUALITY
         )
+        self.config_entry_id = entry_id
```

Comment:
You already have access as `self.platform.config_entry.entry_id`

Reply:
great, thx 👍